src/ui/public/styles/variables/bootstrap-mods.less

```diff
@@ -745,20 +737,6 @@
 @breadcrumb-separator:          "/";
 
 
-//== Carousel
-//
-//##
-
-@carousel-text-shadow:                  0 1px 2px rgba(0,0,0,.6);
```

Comment:
After removing the carousel classes, there are no more references to these variables so I felt it was safe to remove them.